libcxx/test/libcxx/algorithms/alg.sorting/assert.sort.invalid_comparator.pass.cpp

```diff
@@ -11,7 +11,7 @@
 // REQUIRES: has-unix-headers
 // UNSUPPORTED: c++03, c++11, c++14, c++17
 // UNSUPPORTED: !libcpp-hardening-mode=debug
-// XFAIL: availability-verbose_abort-missing
+// XFAIL: libcpp-hardening-mode=debug && availability-verbose_abort-missing
```

Comment:
Note: while for tests that already require the debug mode this is, strictly speaking, redundant, I think that a) it's slightly easier to make this condition uniform throughout the test suite; and b) this is slightly more robust should that requirement ever be lowered (e.g. if changing this test started to run in the extensive mode, it's easy to forget to update the `XFAIL` condition, which might then take a long time to show up on the CI or might even not be covered by the CI).